pkg/query/args.go

```diff
@@ -276,11 +276,6 @@ func (err *ArgsError) Pretty() string {
 `
 	errStr := err.err.Error()
 
-	var prettyErr types.Prettier
-	if errors.As(err, &prettyErr) {
-		errStr = "\n" + types.PrettyIndent(prettyErr, 4)
```

Comment:
Unfortunately, I didn't comment this. Are you sure there wasn't a reason for it? As in: did you provoke the error and check if printing behavior is the same?